ext/standard/tests/file/file_get_contents_variation7.phpt

```diff
@@ -47,11 +47,20 @@ for($i = 0; $i<count($allDirs); $i++) {
 }
 
 chdir($old_dir_path);
+
+echo "\n*** Done ***\n";
+?>
+--CLEAN--
+<?php
+$mainDir = "fileGetContentsVar7.dir";
+$subDir = "fileGetContentsVar7Sub";
+$absMainDir = __DIR__."/".$mainDir;
+$absSubDir = $absMainDir."/".$subDir;
+$filename = 'FileGetContentsVar7.tmp';
+$absFile = $absSubDir.'/'.$filename;
 unlink($absFile);
 rmdir($absSubDir);
 rmdir($absMainDir);
-
-echo "\n*** Done ***\n";
```

Comment:
Why is the ending "Done" message removed in some of these test files but not others?

Reply:
It is not, git is just generating the diff in a weird way.